package.json

```diff
@@ -62,7 +62,9 @@
     "npm-run-all": "4.1.3",
     "rimraf": "2.6.2",
     "sinon": "4.5.0",
-    "ts-node": "^7.0.1"
+    "ts-lint": "4.5.1",
+    "ts-node": "^7.0.1",
+    "typescript": "^3.5.3"
```

Comment:
specifying an explicit dependency on TypeScript and `ts-lint` since we're using these anyway.